tensorflow/core/kernels/BUILD

```diff
@@ -3976,11 +3984,9 @@ tf_kernel_library(
     name = "reduction_ops",
     gpu_srcs = ["reduction_gpu_kernels.cu.h"],
     prefix = "reduction_ops",
-    deps = MATH_DEPS + [":transpose_functor"] + if_cuda([
-        "@cub_archive//:cub",
-    ]) + if_rocm([
-        "@local_config_rocm//rocm:rocprim",
-    ]),
+    deps = MATH_DEPS + 
+    [":cuda_rocm_shared_hdrs",
```

Comment:
run this through "buildifier" else upstream linter will complain

https://github.com/deven-amd/scripts/blob/master/tensorflow/run_buildifier.sh